### PR TITLE
EPMLABSBRN-236 existing code refactoring - make one smart decision of processing files

### PR DESCRIPTION
**Pull request by RiveriRiv**
[EPMLABSBRN-236](https://jira.epam.com/jira/browse/EPMLABSBRN-236)

**Description**:
- Added common list of converters for parsing csv-files
- Splitted csv parser by series

### CODE REVIEW

File: src/main/kotlin/com/epam/brn/service/InitialDataLoader.kt [hunk 1]
```diff
@@ -277,47 +211,26 @@ class InitialDataLoader(
...
+            series = seriesService.findSeriesForId(2L),
             name = "Распознование последовательности слов",
             description = "Распознование последовательности слов",
             template = "<OBJECT OBJECT_ACTION>",
             exerciseType = ExerciseTypeEnum.WORDS_SEQUENCES.toString(),
             level = 1
         )
 
-        // todo: remove this code after making task of loading this data from file
-        task2.exercise = exercise2
```
Comment (ElenaSpb) [inline src/main/kotlin/com/epam/brn/service/InitialDataLoader.kt]: could you please not delete this temporary piece of code that loading data for 2 and 3 series here until their parsers are not written

File: src/test/kotlin/com/epam/brn/service/parsers/csv/CSVParserServiceTest.kt [hunk 2]
```diff
@@ -16,7 +16,7 @@ private inline fun <reified T> makeIdentityConverter(): Converter<T, T> =
         override fun convert(source: T) = source
     }
 
-val csvParserService = CSVParserService()
+val csvParserService = FirstSeriesCSVParserService()
```
Comment (ElenaSpb) [inline src/test/kotlin/com/epam/brn/service/parsers/csv/CSVParserServiceTest.kt]: could you also rename the test please 
FirstSeriesCSVParserServiceTest ?

File: src/main/kotlin/com/epam/brn/service/InitialDataLoader.kt (hunk 1 above)
Comment (RiveriRiv) [reply src/main/kotlin/com/epam/brn/service/InitialDataLoader.kt]: done

File: src/test/kotlin/com/epam/brn/service/parsers/csv/CSVParserServiceTest.kt (hunk 2 above)
Comment (RiveriRiv) [reply src/test/kotlin/com/epam/brn/service/parsers/csv/CSVParserServiceTest.kt]: done

File: src/main/kotlin/com/epam/brn/service/parsers/csv/converter/impl/ExerciseCsvToExerciseModelConverter.kt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+        return target
+    }
+
+    private fun convertSeries(source: ExerciseCsv, target: Exercise) {
+        target.series = seriesService.findSeriesForId(source.seriesId)
+    }
+
+    private fun convertExerciseType(source: ExerciseCsv, target: Exercise) {
+        val exerciseType =
```
Comment (goopnigoop) [inline src/main/kotlin/com/epam/brn/service/parsers/csv/converter/impl/ExerciseCsvToExerciseModelConverter.kt]: just fyi

   val exerciseType =
            when (source.seriesId) {
                1L -> ExerciseTypeEnum.SINGLE_WORDS
                else -> ExerciseTypeEnum.WORDS_SEQUENCES
            }

**Pull request merged**